### DISCUSSION TITLE
upload images? gimana caranya?

### DISCUSSION THREAD

**Discussion by reopucino** [gimid/warga#7] [General]
Okeh, sejauh ini belum ada upload media list untuk gimid sendiri, atau saya yang belum tau cara uploadnya?

Tetapi upload yang lama sudah ada dan sudah terbentuk linknya juga.

**Comment by igrir** [comment]
Buat upload image bisa langsung drag and drop saja om ke text editornya. Copy paste (ctrl+c ctrl+v) di editornya langsung juga bisa
![image](https://github.com/gimid/warga/assets/613950/571f2c50-d39c-4234-88c2-9333e7aae08d)

Hm... sepertinya ketika ngeklik tombol gambar lebih baik langsung muncul file chooser aja ya?

**Reply by reopucino** [reply]
bisa seperti itu, atau juga info semacam "drag aja gambarnya kemareh" pilih yang menurut om giri lebih mudah dulu aja.

**Reply by igrir** [reply]
oh, pas diklik tombol masukin gambarnya gitu ya muncul tulisan begitu? itu menarik juga ✍️ 

buat yang nulis di hape atau tablet sih bakal bermasalah, mereka kayaknya butuh file picker

**Reply by reopucino** [reply]
apakah dengan "drag gambar" nulis yang di hape lebih baik? Saya rasa experiencenya akan sama. Mana yang lebih sulit untuk pengguna HP, drag gambar, atau file picker?

Saya hampir ga pernah nulis di HP untuk blogging. Jadi minim pengalaman disana.

**Reply by igrir** [reply]
Iya sih ya, buat penulis kayaknya jarang untuk menulis di hape.

Btw ternyata ini solusi menarik dari github, di depan mata banget 😂
![image](https://github.com/gimid/warga/assets/613950/bf481972-ba2f-416e-9587-4d0526bdc18f)

kalau diklik dia jadi file picker

**Reply by reopucino** [reply]
nah kan.. berarti hal seperti ini lumayan berpengaruh.. hahahaha. UX seperti ini yang saya sering banget terlewat.
